go.work

```diff
@@ -19,6 +19,7 @@ use (
 	./op-proposer
 	./op-service
 	./proxyd
+	./signer
```

Comment:
Should this be `op-signer` to keep in line with the other protocol related services?

Reply:
I considered that but didn't want to confuse people that it was part of the op-stack or something. Not sure if thats the right way to think about it but happy to change it.